arch/riscv/include/asm/cove_promote.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Definitions for promotion of VM to TVM.
+ *
+ * Copyright (c) 2024 IBM Corp.
+ *
+ * Authors:
+ *     Wojciech Ozga <woz@zurich.ibm.com>
+ */
+
+#ifndef __RISCV_COVE_PROMOTE_H__
+#define __RISCV_COVE_PROMOTE_H__
+
+#ifdef CONFIG_RISCV_COVE_GUEST_PROMOTE
+#define COVE_PROMOTE_SBI_EXT_ID 0x434F5648
```

Comment:
We don't need this header file at all. This is the COVH extension and PROMOTE_TVM FID which shouldn't reused directly this way. I know you can't include the sbi.h here. That's probably why you added another header file with just two EID/FID definitions. But I don't think we should promote a TVM this way at all.

Reply:
Indeed I had this header because I couldn't reuse `sbi.h`. Following your advice I removed it. I use the constants directly in the head.S and commented their meaning/reference to constants in sbi.h.